Backend/test.py

```diff
@@ -18,10 +59,56 @@
 DisplayPort over USB-C - USB-C docking station links all devices to the laptop using a single USB-C cable. Extend the traditional USB capabilities with native multi-display video, audio, data, and power delivery (on select Dell laptops) to charge the laptop with a single USB-C cable. The reversible connector is convenient to use USB-C, with no wrong orientation.
 USB 3.0 - Universal docking stations work with DisplayLink technology that enables docking features over USB. The universal docking solution enables multiple monitors, audio, Ethernet, and other USB devices to be connected to laptops through USB. Works best with USB 3.0 ports on the laptop.
 WiGig (wireless) - WiGig or wireless docking stations work with a laptop that is configured with a WiGig adapter. Only select Dell laptops support WiGig technology."""
+    # construct post request 
+    # post_url = f"{endpoint}/language/authoring/analyze-text/projects/{project_name}/:import?api-version={api_version}"
+    post_url = "https://dellsemanticproductlink.cognitiveservices.azure.com/language/analyze-text/jobs?api-version=2022-10-01-preview"
+    # GET request to get the status of your importing your project
+    get_url = "https://dellsemanticproductlink.cognitiveservices.azure.com/language/analyze-text/jobs?api-version=2022-10-01-preview"
+    # Request body 
+    with open("/Users/zhangc/Documents/CS Year 2/Semester 2/CSU22013 Software Engineering Project I /SwEng-Group-14-SemanticProductLink/Training_Data/weijian.json", "r") as f:
+        requests_body = json.load(f)
 
-@app.route("/")
-def hello_world():
-    return f"<p>{test_data}</p>"
+    # Make POST request to Azure model endpoint 
+    
+    
+    # Possible error scenarios for this request:
+    # The selected resource doesn't have proper permissions for the storage account.
+    # The storageInputContainerName specified doesn't exist.
+    # Invalid language code is used, or if the language code type isn't string.
+    # multilingual value is a string and not a boolean.
+    # if request.method == "POST": 
+    response = requests.post(url=post_url, headers=headers, json=requests_body)
```

Comment:
It looks like you are posting the training data instead of your test data to the analyze text endpoint? I've had a look at this https://learn.microsoft.com/en-us/azure/ai-services/language-service/custom-named-entity-recognition/how-to/call-api?tabs=rest-api and it looks like you have to 

1. build a request object as per the documentation
2. Convert that into a JSON-string (or do this https://stackoverflow.com/a/26344315/323071)
3. POST that to the /jobs endpoint
4. Read the `operation-location` header from the response (which is a URL)
5. In a loop, make a GET request to the operation location every 1 second until you get a 2xx status code (ignoring other status codes/errors)
6. If the loop reaches 30 tries, give up and return an error

You could do more sophisticated things (like using https://pypi.org/project/retrying/)
disclaimer: haven't tried any of the above

Then proceed with the rest of the processing...

Reply:
I got a successful response code!! Thank you Quinten and Sarah for all the help